docs/paper.md

```diff
@@ -50,57 +50,61 @@ affiliations:
    index: 3
 
 
-date: 12 June 2024
+date: 22 June 2024
 bibliography: paper.bib 
 
 ---
 
 # Summary
 
-`scores` is a Python package containing mathematical functions for the verification, evaluation and optimisation of forecasts, predictions or models. It primarily supports the geoscience communities; in particular, the meteorological, climatological and oceanographic communities. In addition to supporting the Earth system science communities, it also has wide potential application in machine learning and other domains such as economics.
+`scores` is a Python package containing mathematical functions for the verification, evaluation and optimisation of forecasts, predictions or models which use labelled n-dimensional data. It primarily supports the geoscience communities; in particular, the meteorological, climatological and oceanographic communities. `scores` also has wide potential application in any situation where labelled n-dimensional (multidimensional) data is used, including in machine learning. 
```

Comment:
Do we need to mention n-dimensional data in the first sentence, if we are also mentioning it in the last sentence in the paragraph? I'm not sure if this sounds better:

 ```suggestion
`scores` is a Python package containing mathematical functions for the verification, evaluation and optimisation of forecasts, predictions or models. It primarily supports the geoscience communities; in particular, the meteorological, climatological and oceanographic communities. `scores` supports labelled n-dimensional (multi-dimensional) data and can potentially be used in other applicable fields, including machine learning. 
```

Reply:
I think @nikeethr 's change is great and I suggest accepting it as-is now. However I'm tagging @nicholasloveday might want to come and check on this later as he might also have some post-hoc feedback to give.

Reply:
What about something along the lines of:

"`scores` is a Python package containing mathematical functions for the verification, evaluation and optimisation of forecasts, predictions or models. It supports labelled n-dimensional (multidimensional) data, which is used in many scientific fields and machine learning. At present, `scores` primarily supports the geoscience communities; in particular, the meteorological, climatological and oceanographic communities." ?

Reply:
@Steph-Chong  I love it! Please put that into the PR, so everyone can then make comments on that version as they wish.

Reply:
> @Steph-Chong I love it! Please put that into the PR, so everyone can then make comments on that version as they wish.

Done - now added to this PR for review.

Reply:
This change looks good to me.

Reply:
Thanks all. I will now mark this as resolved.